src/Toolkit/Toolkit/UI/Controls/Bookmarks/Bookmarks.Windows.cs

```diff
@@ -0,0 +1,177 @@
+﻿// /*******************************************************************************
+//  * Copyright 2012-2018 Esri
+//  *
+//  *  Licensed under the Apache License, Version 2.0 (the "License");
+//  *  you may not use this file except in compliance with the License.
+//  *  You may obtain a copy of the License at
+//  *
+//  *  http://www.apache.org/licenses/LICENSE-2.0
+//  *
+//  *   Unless required by applicable law or agreed to in writing, software
+//  *   distributed under the License is distributed on an "AS IS" BASIS,
+//  *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  *   See the License for the specific language governing permissions and
+//  *   limitations under the License.
+//  ******************************************************************************/
+
+#if !XAMARIN
+using System.Collections.Generic;
+using Esri.ArcGISRuntime.Mapping;
+using Esri.ArcGISRuntime.UI.Controls;
+#if NETFX_CORE
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+#else
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+#endif
+
+namespace Esri.ArcGISRuntime.Toolkit.UI.Controls
+{
+#if !__NETFX_CORE__
+    [TemplatePart(Name = "List", Type = typeof(ListView))]
+#endif
+    public partial class Bookmarks
+    {
+        private void Initialize() => DefaultStyleKey = typeof(Bookmarks);
+
+#if NETFX_CORE
+        protected override void OnApplyTemplate()
+#else
+        public override void OnApplyTemplate()
+#endif
+        {
+            base.OnApplyTemplate();
+            Refresh();
+        }
+
+        internal void Refresh()
+        {
+            ListView = GetTemplateChild("List") as ListView;
+            if (ListView == null)
+            {
+                return;
+            }
+
+            ListView.ItemsSource = GetCurrentBookmarkList();
```

Comment:
This should be done in the control template instead with `{TemplateBinding}` (so you can also remove the property below)